src/go/util/httptest_util.go

```diff
@@ -71,3 +73,18 @@ func InitMockServerFromPathResp(pathResp map[string]string) *httptest.Server {
 		w.WriteHeader(http.StatusNotFound)
 	}))
 }
+
+// JsonEqual compares two JSON strings after normalizing them.
+func JsonEqual(x, y *string) bool {
```

Comment:
do we want to be consistent on the calling order; first one is "got", the second is "want"?  for consistency maybe?

Reply:
Done.